resolwe_bio/processes/support_processors/multiqc.yml

```diff
@@ -42,7 +41,6 @@
         - name: config
           label: Use configuration file
           type: basic:boolean
-          default: false
```

Comment:
Isn't this just the opposite of what the changelog message says? If it is used by defalt shouldn't you change to `deafult: true`?

Reply:
`default: true` is redundant, since `default: true` is the default state when variable is not specified explicitly.